packages/react-components/react-dialog/library/src/components/DialogBackdropMotion.ts

```diff
@@ -1,16 +1,9 @@
 import { createPresenceComponent, motionTokens } from '@fluentui/react-motion';
+import { createFadePresence } from '@fluentui/react-motion-components-preview';
 
-const keyframes = [{ opacity: 0 }, { opacity: 1 }];
-
-export const DialogBackdropMotion = createPresenceComponent({
-  enter: {
-    keyframes,
-    easing: motionTokens.curveLinear,
-    duration: motionTokens.durationGentle,
-  },
-  exit: {
-    keyframes: [...keyframes].reverse(),
-    easing: motionTokens.curveLinear,
-    duration: motionTokens.durationGentle,
-  },
-});
+export const DialogBackdropMotion = createPresenceComponent(
+  createFadePresence({
+    enterDuration: motionTokens.durationGentle,
+    enterEasing: motionTokens.curveLinear,
+  }),
+);
```

Comment:
```suggestion
export const DialogBackdropMotion = FadeRelaxed;
```
The solution is simple: use the appropriate `Fade` variant. There's no need to create a one-off fade.